Makefile

```diff
@@ -1,9 +1,10 @@
 # cobbled together really fast
 CC = gcc
 
-# no matter how hard I try, I can't get make to accept that I'm on windows.
-ifneq ($(shell echo),)
-  DOTEXE=.exe
+# Make an exe if on windows
+ifeq ($(OS),Windows_NT)
+	DOTEXE=.exe
+	RM = del
```

Comment:
Oddly enough, this line is giving me issues when trying to run `make clean` through Windows command prompt/`cmd.exe`. It's not really reasonable for me to expect people to install `rm` either... :/

Reply:
I see that OS variable set and checked like that for a lot of cross compiled projects. I was using CMDER and have a few MinGW toolchains installed. Maybe one of them is setting it. I just opened a clean cmd on windows 10 and executed:
echo %OS%
and Windows_NT came back. I'm sure it's not set for all versions of windows.

Reply:
I too have "Windows_NT" display when doing `echo %OS%` manually (at least in `cmd.exe`).

Doing `echo $OS` when in bash provides the same thing; I don't know why swapping out `rm` for `del` would be affected though...

Reply:
Got it working but what a hack, haha. Ok, it's there if someone wants it. Feel free to discard.